aries-site/src/examples/components/tip/TipExitExample.js

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
-import { Box, Button, Tip, Text } from 'grommet';
+import { Box, Button, Paragraph, Tip, Text } from 'grommet';
 import { FormClose } from 'grommet-icons';
 
 export const TipExitExample = () => (
-  <Box gap="small" direction="row">
+  <Box gap="small" direction="row" align="start">
     <Box width="small">
       <Text color="text-strong" size="large" weight="bold">
         Add service
       </Text>
-      <Text>
-        Visit the HPE catalog to start addiing all of your favoriite services.
-      </Text>
-    </Box>
-    <Box>
-      <Tip content="Close">
-        <Button a11yTitle="close" icon={<FormClose />} />
-      </Tip>
+      <Paragraph margin="none">
+        Visit the HPE catalog to start addiing all of your favorite services.
+      </Paragraph>
     </Box>
+    <Tip content="Close">
+      <Button a11yTitle="close" icon={<FormClose />} />
```

Comment:
```suggestion
      <Button a11yTitle="close" tip="Close" icon={<FormClose />} />
```
While we're touching this, can we simplify to this and remove Tip wrapper?

Reply:
done.